include/abb_librws/rws_common.h

```diff
@@ -493,6 +518,11 @@ struct SystemConstants
        * \brief State.
        */
       static const std::string STATE;
+
+      /**
+       * \brief Sys.
```

Comment:
Is there really no description of this?

Reply:
Oops, I forgot this one.

It is the `controller` topic in the configurations, which is abbreviated as `sys`.